tokens/src/core/global/typography.json

```diff
@@ -2,90 +2,284 @@
-        "lg": { "value": "1.4063rem", "type": "dimension", "source": "$font-size-lg", "description": "Large font size." },
-        "sm": { "value": ".875rem", "type": "dimension", "source": "$font-size-sm", "description": "Small font size." },
-        "xs": { "value": ".75rem", "type": "dimension", "source": "$font-size-xs", "description": "Extra-small font size." },
-        "small": {
-          "base": { "value": "87.5%", "type": "percentage", "source": "$small-font-size", "description": "Small font size." },
-          "x": { "value": "75%", "type": "percentage", "source": "$x-small-font-size", "description": "X-small font size." }
-        },
-        "h1": { "value": "2.5rem", "type": "dimension", "source": "$h1-font-size", "description": "Font size of heading level 1." },
-        "h2": { "value": "2rem", "type": "dimension", "source": "$h2-font-size", "description": "Font size of heading level 2." },
-        "h3": { "value": "1.375rem", "type": "dimension", "source": "$h3-font-size", "description": "Font size of heading level 3." },
-        "h4": { "value": "1.125rem", "type": "dimension", "source": "$h4-font-size", "description": "Font size of heading level 4." },
-        "h5": { "value": ".875rem", "type": "dimension", "source": "$h5-font-size", "description": "Font size of heading level 5." },
-        "h6": { "value": ".75rem", "type": "dimension", "source": "$h6-font-size", "description": "Font size of heading level 6." },
-        "mobile": {
-          "h1": { "value": "2.25rem", "type": "dimension", "source": "$h1-mobile-font-size", "description": "Mobile font size of heading level 1." },
-          "h2": { "value": "{typography.font.size.h2}", "type": "dimension", "source": "$h2-mobile-font-size", "description": "Mobile font size of heading level 2." },
-          "h3": { "value": "{typography.font.size.h3}", "type": "dimension", "source": "$h3-mobile-font-size", "description": "Mobile font size of heading level 3." },
-          "h4": { "value": "{typography.font.size.h4}", "type": "dimension", "source": "$h4-mobile-font-size", "description": "Mobile font size of heading level 4." },
-          "h5": { "value": "{typography.font.size.h5}", "type": "dimension", "source": "$h5-mobile-font-size", "description": "Mobile font size of heading level 5." },
-          "h6": { "value": "{typography.font.size.h6}", "type": "dimension", "source": "$h6-mobile-font-size", "description": "Mobile font size of heading level 6." }
+        "$type": "dimension",
+        "base": {
+          "source": "$font-size-base",
+          "$value": "1.125rem",
+          "$description": "Base font size."
         },
-        "lead": {
-          "value": "calc({typography.font.size.base} * 1.25)",
-          "type": "dimension",
+        "lg": {
           "source": "$lead-font-size",
-          "description": "Lead text font size."
+          "$value": "{typography.font.size.base} * 1.25",
```

Comment:
D'oh, I think in my commit I lost the `calc(...)` that was previously wrapping this token's `$value` (this was the only token I experimented with). When previewing the docs site locally, the `.lead` utility class isn't working properly anymore, noticed first on the home page. Adding `calc(...)` back in will resolve.

[context] This was removed while exploring the `sd-transforms` transform for `ts/resolveMath` to see if we could abstract away `calc`, but as discussed in another comment, there is not currently a great solution for this, so we need to keep `calc` in here, at least for now :)

Reply:
[inform] Implemented in follow-up PR: https://github.com/openedx/paragon/pull/3222